docs/advanced_usage.md

```diff
@@ -452,7 +455,8 @@ services:
 ```
 
 1. Note the `:ro` tag means the file will be mounted as read only.
-2. `flower` runs by default on port 5555, but this can be configured
+2. Flower runs by default on port 5555, but this can be configured.
+3. Start Flower by the container.
```

Comment:
```suggestion
```

this line doesn’t really make sense (I mean grammatically) and seems sort of obvious I think. i would just remove it.